web/scss/pages/_ancillary.scss

```diff
@@ -8,17 +8,45 @@
   h2,
   h3,
   h4,
-  h5,
+  h5 {
+    + blockquote,
+    + hr,
+    + iframe,
+    + img,
+    + pre,
+    + video,
+    + .button {
+      margin-top: 0.25rem;
+    }
+
+    + footer,
+    + form,
+    + h1,
+    + h2,
+    + h3,
+    + h4,
+    + h5,
+    + ol,
+    + p,
+    + ul {
+      padding-top: 0.25rem;
```

Comment:
Why not all margin? You want the margin collapse behavior on the top elements but not the bottom ones?

Reply:
Emailing hello@lbry.com will work best.